pkgs/development/python-modules/pytomlpp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pybind11
+, pytestCheckHook
+, dateutil
+, doxygen
+, python
+, pelican
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "pytomlpp";
+  version = "0.3.5";
+  # pelican needs > 2.7
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "bobfang1992";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "1h06a2r0f5q4mml485113mn7a7585zmhqsk2p1apcybyydllcqda";
+  };
+
+  buildInputs = [ pybind11 ];
+
+  checkInputs = [
+    pytestCheckHook
+
+    dateutil
+    doxygen
+    python
+    pelican
+    matplotlib
+  ];
+
+  preCheck = ''
+    cd tests
+  '';
+
+  pythonImportsCheck = [ "pytomlpp" ];
+
+  meta = with lib; {
+    description = "A python wrapper for tomlplusplus";
+    homepage    = "https://github.com/bobfang1992/pytomlpp";
+    license     = licenses.mit;
+    maintainers = with maintainers; [ evils ];
```

Comment:
A lot of maintainers are using `nixkgs-fmt` to do the formatting. With the first run the style of `meta` would be updated probably if somebody else is working on the package.

Reply:
this style was copied from another file i used as a template, will remove it